views/admin/updatePost.ejs

```diff
@@ -0,0 +1,15 @@
+<h1>Admin Panel</h1>
+<h2>Update Posts</h2>
+
+<form action="/admin/posts/updatePost" method="patch" class="submit-form">
+  <label for="name">Author Name:</label>
```

Comment:
get alll the information of the current post which is need to be updated, then make changes and send update request